test/scraping/cycling/procyclingstats/raceStageClassifications.test.js

```
@@ -1,84 +1,252 @@
+          `${TEST_DATA_DIR}/raceStageClassificationMountain.csv`,
+        ).delete();
+        await Bun.file(
+          `${TEST_DATA_DIR}/raceStageClassificationPoints.csv`,
+        ).delete();
+        await Bun.file(
+          `${TEST_DATA_DIR}/raceStageClassificationTeams.csv`,
+        ).delete();
+        await Bun.file(
+          `${TEST_DATA_DIR}/raceStageClassificationYouth.csv`,
+        ).delete();
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)